### DISCUSSION TITLE
Build edk2

### DISCUSSION THREAD

**Discussion by tushar95mahawar** [community/events#56] [Community]
Hi,
I'm facing problem to build edk2 environment.
I'm giving command c:\edk2\> build and getting error: 000E: File directory not found in workspace c:\edk2\MdeModulePkg\Library\BrotliCustomDecompressLib\brotli\c\include

**Comment by Akash1134** [comment]
Hi 👋  @tushar95mahawar Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we’re glad you’re here!

We want to make sure you’re getting the best support you can, If you have questions about the code or run into obstacles getting things to work, please join the EDK II developer [Mailing-Lists](https://github.com/tianocore/tianocore.github.io/wiki/Mailing-Lists) and ask your EDK II related questions on the list.

You can also visit their [repo](https://github.com/tianocore/edk2) and [discussions](https://github.com/tianocore/edk2/discussions) for further help.
